packages/gafl-webapp-service/src/handlers/__tests__/attribution-handler.spec.js

```diff
@@ -75,6 +87,54 @@ describe('The attribution handler', () => {
     delete process.env.ATTRIBUTION_REDIRECT
   })
 
+  it('redirects to RENEWAL_BASE if journey is renewal', async () => {
+    const query = {
+      [UTM.CAMPAIGN]: 'renewals',
+      [UTM.MEDIUM]: 'click_bait',
+      [UTM.CONTENT]: 'eieioh',
+      [UTM.SOURCE]: 'tomato',
+      [UTM.TERM]: 'Michaelmas'
+    }
+    const responseToolkit = generateResponseToolkitMock()
+
+    await attributionHandler(generateRequestMock(query), responseToolkit)
+    expect(responseToolkit.redirect).toHaveBeenCalledWith(RENEWAL_BASE)
+  })
+
+  it('redirects begins with { RENEWAL_BASE } when journey is renewal', async () => {
+    const query = {
+      [UTM.CAMPAIGN]: 'renewals',
+      [UTM.MEDIUM]: 'click_bait',
+      [UTM.CONTENT]: 'eieioh',
+      [UTM.SOURCE]: 'tomato',
+      [UTM.TERM]: 'Michaelmas'
+    }
+    const responseToolkit = generateResponseToolkitMock()
+    const regExMatch = new RegExp(`${RENEWAL_BASE}`)
```

Comment:
This RegExp needs to be anchored to the start of the string, otherwise it'll just be checking that RENEWAL_BASE occurs somewhere in the url (which might give us a passing test, but ultimately not testing what we need it to).
Change it to 
```suggestion
    const regExMatch = new RegExp(`^${RENEWAL_BASE}`)
```